src/uct/ib/base/ib_device.h

```diff
@@ -96,7 +96,9 @@ enum {
     UCT_IB_ADDRESS_FLAG_SUBNET64       = UCS_BIT(3),
     /* Used for IB link layer */
     UCT_IB_ADDRESS_FLAG_IF_ID          = UCS_BIT(4),
-    UCT_IB_ADDRESS_FLAG_LAST           = UCS_BIT(5)
+    /* Defines active MTU size */
```

Comment:
rename MTU -> PATH_MTU
this means it's not the MTU of the remote side, but rather the **path** MTU